sponsors.md

```diff
@@ -101,17 +100,18 @@ title: Sponsoring
 
 </div>
 
+-->
+{% endcomment %}
 
 <section class="whitewithwheel">
   <h2>Interested in sponsoring us?</h2>
   <br />
   <p>
-    <a class="button" href="mailto:sponsors@rustfest.eu">
+    <a class="button" href="mailto:team@rustfest.eu">
```

Comment:
enabled back the block for sponsorship packages and commented out the HTML that we don't need to expose (atm)